config/template.go

```diff
@@ -48,7 +82,12 @@ func defaultValue(defaultValue, templateValue interface{}) string {
 func NewTemplate(config []byte) (*Template, error) {
 	env := parseEnvironment(os.Environ())
 	tmpl, err := template.New("").Funcs(template.FuncMap{
-		"default": defaultValue,
+		"default":       defaultValue,
+		"split":         split,
+		"join":          join,
+		"replace":       replace,
+		"regexReplace":  regexReplace,
+		"regexpReplace": regexReplace,
```

Comment:
> Why the alias? 

Um... just because? In all honesty, my first run of tests failed because I made that mistake! Seemed easiest to fix it by aliasing.

I don't care much one way or the other. If we want to use template's `replaceAll`, `regexMatch`, `regexReplaceAll`, I am fine with that. Whatever you want, just let me know,  happy to modify, rerun tests and push out.